schema.sql

```diff
@@ -1 +1,8 @@
-CREATE DATABASE goskeleton;
+CREATE TABLE "USER"
```

Comment:
Oh boy. It's quite a big topic for discussing version control for databases https://blog.codinghorror.com/get-your-database-under-version-control/ But here should be also `CREATE DATABASE goskeleton;`

Reply:
Not sure if using `CREATE DATABASE goskeleton;` is a good idea. At first it doesn't seem to be possible to create and modify the same database in one script in postgres. Second is hardcoded db name won't be convenient in case of separate databases for development and testing on one  machine.